server/recruitment.py

```diff
@@ -89,24 +98,78 @@ def edit_applicant_notes(applicant_user_id, text):
         applicant.notes = text
         return {'status': 'ok'}
 
+def get_user_list():
+    # list of all Users with roles
+    result = {}
+    for user in User.query().run():
+        id = user.get_id()
+        user_name = Character.get(id).name if id else None
+        result[id] = {
+            'user_id': id,
+            'is_recruiter': user.is_recruiter,
+            'is_snr_recruiter': user.is_senior_recruiter,
+            'is_admin': user.is_admin,
+            'name': user_name,
+        }
+    return result
+
+def get_character_search_list(search_text):
+    # list of all chars with name beggining with search_text
+    result = {}
+    for character in Character.query().\
```

Comment:
FYI I don't know enough to confirm that this query will work. Fine with putting it here but keep that in mind if search isn't working.

Reply:
it should, as it's its a range search on a string index, but you'll be SQLing soon